main/ejml-dsparse/src/org/ejml/sparse/csc/RandomMatrices_DSCC.java

```diff
@@ -45,18 +44,21 @@
      * @param rand Random number generator
      * @return Randomly generated matrix
      */
-    public static DMatrixSparseCSC rectangle(int numRows , int numCols , int nz_total ,
-                                             double min , double max , Random rand ) {
+    public static DMatrixSparseCSC rectangle( int numRows, int numCols, int nz_total,
+                                              double min, double max, Random rand ) {
+
+        if (UtilEjml.exceedsMaxMatrixSize(numRows, numCols))
+            throw new IllegalArgumentException("You have specified a matrix which exceeds the size of an integer");
```

Comment:
this more like for temporary fix for the random sparse matrix right?
Maybe we also want to assert, that `nz_total` has a positive value?

Reply:
So maybe we could change the error message to something like `This method cannot create random sparse matrices, where the dense version would not fit`